lib/pypechain/pypechain/run_pypechain.py

```diff
@@ -0,0 +1,116 @@
+"""Script to generate typed web3.py classes for solidity contracts."""
+from __future__ import annotations
+
+import json
+import os
+import sys
+from pathlib import Path
+
+from jinja2 import Environment, FileSystemLoader
+from pypechain.utilities import avoid_python_keywords, is_abi_function, solidity_to_python_type
+from web3 import Web3
+from web3.types import ABIFunction
+
+
+def load_abi_from_file(file_path: Path):
```

Comment:
might as well add proper docstrings on all of these funcs. I think we're going to turn on that linter soon.

Reply:
100%!